libxml2 2.10.x deprecation warnings xmlSchemaCleanupTypes

## Expected behavior and actual behavior.

I expect that MapServer pay attention to the change in current 2.10.x version of libxml2, released 1 month ago.

```
make[3]: *** [CMakeFiles/mapserver.dir/build.make:64: CMakeFiles/mapserver.dir/fontcache.c.o] Error 1
make[3]: *** Waiting for unfinished jobs....
//mapserver/mapowscommon.c:713:5: warning: 'xmlSchemaCleanupTypes' is deprecated [-Wdeprecated-declarations]
    xmlSchemaCleanupTypes();
    ^
/dist/include/libxml2/libxml/xmlschemastypes.h:35:1: note: 'xmlSchemaCleanupTypes' has been explicitly marked deprecated here
XML_DEPRECATED
^
/dist/include/libxml2/libxml/xmlversion.h:464:43: note: expanded from macro 'XML_DEPRECATED'
#    define XML_DEPRECATED __attribute__((deprecated))
                                          ^
1 warning generated.
1 warning generated.
6 warnings generated.
make[2]: *** [CMakeFiles/Makefile2:96: CMakeFiles/mapserver.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:103: CMakeFiles/mapserver.dir/rule] Error 2
make: *** [Makefile:164: mapserver] Error 2
```

https://github.com/GNOME/libxml2/commit/40483d0ce26119fd3b23aea6cce532a8a78670cf

## Steps to reproduce the problem.

Compile with most recent libxml2 2.10.x dependency.

## MapServer version and installation method

MapServer 7.6.4, but the current 8.0.0 should be get the same deprecation warning

--- a/mapowscommon.c
+++ b/mapowscommon.c
@@ -718,7 +718,6 @@ int msOWSSchemaValidation(const char* xml_schema, const char* xml)
 
   /* If XML Schema hasn't been rightly loaded */
   if (schema == NULL) {
-    xmlSchemaCleanupTypes();
     xmlMemoryDump();
     xmlCleanupParser();
     return ret;

